### DISCUSSION TITLE
Change top left logo background color

### DISCUSSION THREAD

**Discussion by markusmoenig** [McShelby/hugo-theme-relearn#489] [Q&A] [answered]
Hi,

thanks for the excellent theme.

How can I change the background color of the logo ? Did not find it in the docu.

**Comment by McShelby** [comment, marked as answer]
If you have given your own logo in `layouts/partials/logo.html` you can define the preferred styles there.

**Reply by markusmoenig** [reply]
Thanks, right now I have 

```html
<a href="/"><img src="/images/logo.png" alt="logo"></a>
```

in there and it is working fine. How can I change the background color ?

**Reply by McShelby** [reply]
This is a general HTML/CSS question and additionally depends on your logo. You could try to use a CSS `background-color` setting. https://developer.mozilla.org/en-US/docs/Web/CSS/background-color

**Reply by markusmoenig** [reply]
Mmh, ok tried
```html
<style>
    #logo {
        background-color: white;
    }
</style>
<a href="/"><img src="/images/logo.png" alt="logo"></a>
```

but it does not work. Not everybody has any idea about html or css.

**Reply by markusmoenig** [reply]
Ok
```html
<style>
    #header {
        background-color: white;
        color: white;
    }
    #header-wrapper  {
        background-color: white;
        color: white;
    }
</style>
<a href="/"><img src="/images/logo.png" alt="logo"></a>
```

Did the trick, but would it not be easier to make the primary color (the green in the dark style for example) configurable ? That way people could brand their website to their CI / logo. I do not want to create new themes etc.

**Reply by McShelby** [reply]
It is, but it requires a certain knowlegde of how CSS works. https://mcshelby.github.io/hugo-theme-relearn/basics/customization/index.html#roll-your-own

**Reply by McShelby** [reply]
Also, check out the interactive variant generator at https://mcshelby.github.io/hugo-theme-relearn/basics/generator/index.html
